### ISSUE TITLE
💡 Research how to create a basic backend with Express.js

### ISSUE THREAD

**Issue by Thenlie** [Thenlie/Streamability#41]
# Backend

An initial backend needs to be created with Express.JS to make this a full stack application. The initial creation with Vite is front-end only. 

## Description

Research will need to be done to figure out the best way to host this. Ideally the development process will be seamless as with [concurrently](https://www.npmjs.com/package/concurrently). Similarly, the deployment process should also be close to seamless. The hope would be that the deployment can be bundled into one but that may not be possible. 

## TL:DR

The inital goal is to simply research what the possible options are and then start a discussion with them.

**Comment by Thenlie** [comment]
## Resources

- https://vercel.com/blog/monorepos
- https://vercel.com/docs/concepts/functions/serverless-functions/runtimes
- https://github.com/adiun/vite-monorepo

**Comment by Thenlie** [comment]
https://supabase.com/
